app/services/reporting/agency_and_sp_report.rb

```diff
@@ -30,14 +35,12 @@ def agency_and_sp_emailable_report
     end
 
     def active_agencies
-      @active_agencies ||= begin
-        Agreements::PartnerAccountStatus.find_by(name: 'active').
-          partner_accounts.
-          includes(:agency).
-          where('became_partner <= ?', report_date).
-          map(&:agency).
-          uniq
-      end
+      @active_agencies ||= Agreements::PartnerAccountStatus.find_by(name: 'active').
```

Comment:
this change should be a no-op but I wanted to clean it up while I was already here